docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -98,6 +98,36 @@ comprising the number of shards in the index, the total size of the index in
 bytes, and the maximum number of segments per shard in the index. Defaults to
 `false`, meaning that this information is omitted.
 
+`sort`::
```

Comment:
The main question left from my end here is on what to do about `?verbose=false` requests. For these, the sorting as implemented here will not work out nicely if the request hits and old-version snapshot that doesn't yet track the timestamps in the `RepositoryData` but will instead just sort by name in this case.

My thinking here would be to maybe just not allow pagination and sorting with `?verbose=false` at all for now. Or alternatively, fail these requests if old version snapshots are found in the repo. 
Another option would be falling back to loading the missing information needed for pagination when running `?verbose=false` queries to work around old repositories but I'm not sure this is worth the effort compared to the other two options.

Reply:
> This would only apply if you specify sort, search or size then?

Yes I think just not allowing any of the three for `?verbose=false` would be the plan for option 1.

Reply:
Went with option 1 in this PR now => no pagination allowed with `verbose=false`